### DISCUSSION TITLE
Turn off MacBook display when external monitor connected

### DISCUSSION THREAD

**Discussion by maxemillianian** [MonitorControl/MonitorControl#1171] [Ideas]
![image](https://user-images.githubusercontent.com/22266723/188293172-2e7e394b-b2a8-4d05-b706-de71e34d6241.png)

I see that Lunar has an option to disable the MacBook's screen when connecting an external display. Would it be possible to implement something like that in MonitorControl? It seems like a useful feature for extending the battery if running the Macbook on battery power.

**Comment by waydabber** [comment]
Sure, but Lunar does not actually turn off the macbook display, only mirrors the external display to the macbook display (so you don't have two desktops) and sets its brightness to 0%.

If your goal is to conserve battery life, you can turn the brightness to 0% any time using the brightness slider.

I don't think a similar feature will make into MonitorControl as this is outside of the scope of the app (the consensus among maintainers is that the app is already a bit bloated if measured against the original vision/scope, there should be a simplification in the future instead so the app will be easier to maintain and more maintainers can join in), but will add it to [BetterDisplay](https://github.com/waydabber/BetterDisplay/issues/474) eventually.

**Comment by maxemillianian** [comment]
Ah okay, sure; I'll just stick to 0% brightness then. Thanks!
